products/centreon.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Centreon
+category: server-app
+tags: php-runtime
+permalink: /centreon
+versionCommand: cat /etc/centreon/centreon.conf
+releasePolicyLink: https://docs.centreon.com/docs/releases/lifecycle/
+releaseDateColumn: true
+activeSupportColumn: true
+eolColumn: Security Support
+
+#identifiers:
+#-   repology: zabbix
+#-   purl: pkg:brew/zabbix
+#-   purl: pkg:github/zabbix/zabbix
+
+auto:
+-   git: https://github.com/centreon/centreon.git
+
+# For non-LTS releases :
```

Comment:
According to https://docs.centreon.com/docs/releases/lifecycle/ there are two phases of 12 months each for versions after 21.10. And I did not see any trace of LTS releases in this doc.

Are you sure of this ?

Reply:
> Are you sure of this ?

I have to check that 👍